Bottom: 95460fe5374d44c039db936540a7cc68bbf1716c
Top:    419e1178d6afd6c9af832dcb47c8bc4a9a66f80f
Author: Edward "Teddy" Brow <edward.teddy.brow@gmail.com>
Date:   2017-04-21 16:18:56 -0700

Refresh of 01_14

---

diff --git a/01_13/main.cc b/01_13/main.cc
new file mode 100644
index 0000000..5c7181f
--- /dev/null
+++ b/01_13/main.cc
@@ -0,0 +1,17 @@
+#include <iostream>
+
+int main()
+{
+  int sum = 0;
+  for (int i = 50; i <= 100; ++i)
+    sum += i;
+
+  std::cout << "sum of numbers in range [50, 100]: " << sum << std::endl;
+
+  std::cout << "countdown from 10 to 0:" << std::endl;
+
+  for (int i = 10; i >= 0; --i)
+    std::cout << i << std::endl;
+
+  return 0;
+}
